dietpi/dietpi-software

```diff
@@ -30,6 +30,34 @@ Available commands:
 	G_CHECK_ROOTFS_RW
 	G_INIT
 	# Import DietPi-Globals ---------------------------------------------------------------
+  
+  # Configuration Creation
+  CONFIG_CREATE(){
+    local file=$1 
+    local service=$2 
+    local timeout=${3:-15} 
+    local content=$CONFIG_CREATE_CONTENT 
+    unset -v CONFIG_CREATE_CONTENT 
+    if [[ -f $file ]] && { [[ ! $content ]] || grep -q "$content" "$file"; } && return 0
+		then
```

Comment:
Did you review the test installs output? Because this can actually not work. The condition itself contains a return, hence the config file is never pre-created 🙂. Let me help with the syntax.